tests/fixtures/api.py

```diff
@@ -15,8 +15,10 @@ async def client(app):
     """
     Yield a test client for testing the api
     """
-
-    async with httpx.AsyncClient(app=app, base_url="https://test/api") as async_client:
+    transport = ASGITransport(app=app)
+    async with httpx.AsyncClient(
+        transport=transport, base_url="https://test/api"
+    ) as async_client:
```

Comment:
Main change is to specify a transport explicitly instead of using the `app` kwarg.